### PR TITLE
Allow events to propagate when disabled.

### LINKED ISSUE #11: Overlapping mouse/touch events
Please add 'enable' and 'disable' functionality to RayController's mouse/touch event binding, overlapping DOM elements won't receive these events.

### PR DESCRIPTION
**Pull request by jasonfarrell**
Closes #11

### CODE REVIEW

File: src/ray-controller.js [hunk 1]
```diff
@@ -90,10 +90,10 @@ export default class RayController extends EventEmitter {
 
     } else {
       // If there's no gamepad, it might be Cardboard, magic window or desktop.
-      if (isMobile()) {
+      if (isMobile() || this.forceVRInteraction) {
```
Comment (borismus) [inline src/ray-controller.js:93]: Where is this defined?

File: src/ray-controller.js [hunk 2]
```diff
@@ -180,22 +180,38 @@ export default class RayController extends EventEmitter {
     this.emit('raydown');
 
     // Prevent synthetic mouse event from being created.
-    e.preventDefault();
+    if (!this.disabled) {
```
Comment (borismus) [inline src/ray-controller.js:183]: Why do we need this?

File: src/ray-controller.js [hunk 3]
```diff
@@ -180,22 +180,38 @@ export default class RayController extends EventEmitter {
...
+      var target = e.target;
+      var isVRBtn = (target.className.length) && (target.className.indexOf('webvr-ui') > -1);
+      $(target).parents().each(function(i, el) {
```
Comment (borismus) [inline src/ray-controller.js:186]: Let's not use $.

File: src/ray-controller.js [hunk 4]
```diff
@@ -180,22 +180,38 @@ export default class RayController extends EventEmitter {
...
+    }
   }
 
   onTouchMove_(e) {
     this.updateTouchPointer_(e);
     this.updateDragDistance_();
 
     // Prevent synthetic mouse event from being created.
-    e.preventDefault();
+    if (!this.disabled) {
```
Comment (borismus) [inline src/ray-controller.js:203]: Where is this.disabled defined?!

**Review by borismus — CHANGES_REQUESTED** [review]
Sorry, I don't understand what you have in mind with this PR. Have you tested? There seem to be are a bunch of undefined variables, etc.

**Comment by borismus** [comment]
Ping?

**Comment by jasonfarrell** [comment]
@borismus I'll close this out. I think the problem we were having is we needed an ability to deactivate the library — events wouldn't propagate.

**Pull request closed without merging**